src/Symfony/Component/Mailer/Bridge/Mailgun/Transport/MailgunApiTransport.php

```diff
@@ -67,10 +67,10 @@ protected function doSendApi(SentMessage $sentMessage, Email $email, Envelope $e
         $result = $response->toArray(false);
         if (200 !== $response->getStatusCode()) {
             if ('application/json' === $response->getHeaders(false)['content-type'][0]) {
-                throw new HttpTransportException(sprintf('Unable to send an email: %s (code %s).', $result['message'], $response->getStatusCode()), $response);
+                throw new HttpTransportException(sprintf('Unable to send an email: "%s" (code "%s").', $result['message'], $response->getStatusCode()), $response);
             }
 
-            throw new HttpTransportException(sprintf('Unable to send an email: %s (code %s).', $response->getContent(false), $response->getStatusCode()), $response);
+            throw new HttpTransportException(sprintf('Unable to send an email: "%s" (code "%s").', $response->getContent(false), $response->getStatusCode()), $response);
```

Comment:
These changes were suggested by fabbot

Reply:
`getStatusCode()` is an integer, so we should replace `(code "%s" by %d)`.
The quotes around the response content should be removed as well IMHO.
The new fabbot rule is far from perfect :)

Reply:
see #36170

Reply:
Thank you. 

I like the idea of the new rule. I've been stuck countless of times with error messages like: 

> "Could not find value for whatever. "

When in fact it it should be: 

> "Could not find 'value' for whatever. "

------

The PR is updated.